src/dom.js

```diff
@@ -530,6 +530,40 @@ export function getDataParamsFromAttributes(element, opt_computeParamNameFunc,
 }
 
 /**
+ * Returns link element url components as url parameters key-value pairs.
+ * e.g. <a href="https://example.com/index.html?query#hash"
+ * -> {
+ *   clickHostname: example.com,
```

Comment:
Would it be better to call these anchorHostname, anchorUrl and so on? The navigation could easily be caused by tap instead of click :).